File: src/main/java/it/tylframework/data/mongo/basics/NumeratorFeeder.java
package it.tylframework.data.mongo.basics;

import it.tylframework.data.mongo.common.Footprint;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.Date;

/**
 * Created by mp on 20/11/14.
 */
@Data
@AllArgsConstructor
public class NumeratorFeeder{
    Boolean freezed;
    Date starting_date;
    Integer starting_number;
    Date ending_date;
    Integer warning_number;
    Integer increment_by;
    Integer last_number_used;
    Boolean output_as_string;
    String output_format;
}

File: src/main/java/it/tylframework/data/mongo/basics/NumeratorFeeder.kt
package it.tylframework.data.mongo.basics

import java.util.Date
import it.tylframework.data.mongo.common.Footprint
import it.tylframework.data.mongo.config.TylContext

/**
 * Created by marco on 19/09/14.
 */
data class NumeratorFeeder(
        var freezed:Boolean,
        var startingDate:Date,
        var startingNumber:Int,
        var endingDate:Date,
        var warningNumber:Int,
        var incrementBy:Int,
        var lastNumberUsed:Int,
        var outputAsString: Boolean,
        var ouputFormat:String)
:Footprint(TylContext.currentUser, Date() ,TylContext.currentUser,  Date()){

}